docs/openapi/components/schemas/common/Invoice.yml

```diff
@@ -396,12 +395,12 @@ example: |-
         }
       }
     },
-    "consignee": {
+    "consignee": [{
```

Comment:
I suppose if we permit intermediates, but will need to signify ultimately who is final